pkg/eventstore/database.go

```diff
@@ -161,6 +161,7 @@ func (t *table) Insert(ctx context.Context, ev components.Event) error {
 	return insertEvent(ctx, t.dbRW, t.table, ev)
 }
 
+// Returns nil if the event is not found.
```

Comment:
if you use goland, you will see there is a lint warning at the comment $^$...  
```suggestion
// Find returns nil if the event is not found.
```